packages/admin-panel-server/src/routes/FetchReportPreviewData.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ *
+ */
+
+import { Request, Response, NextFunction } from 'express';
+
+import { Route } from '@tupaia/server-boilerplate';
+
+import { ReportConnection } from '../connections';
+import { DraftReportExtractor } from '../viz-builder';
+
+export class FetchReportPreviewData extends Route {
```

Comment:
```suggestion
export class FetchReportPreviewDataRoute extends Route {
```
Small thing, but it's a bit of a convetion for us to suffix all Route classes with `Route` 👍